__tests__/app.test.js

```diff
@@ -217,6 +217,46 @@ describe("nc_games_test", () => {
         });
     });
   });
+  describe("/api/reviews/:review_id/comments", () => {
+    it("201: POST /api/reviews/:review_id/comments should return a new comment", () => {
+      return request(app)
+        .post("/api/reviews/3/comments")
+        .send({
+          username: "mallionaire",
+          body: "I love hosting",
+        })
+        .expect(201)
+        .then(({ body }) => {
+          const { comment } = body;
+          expect(comment).toMatchObject({
+            review_id: expect.any(Number),
+            author: expect.any(String),
+            body: expect.any(String),
+            comment_id: expect.any(Number),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+          });
+        });
+    });
+    it("400: malformed body/ missing required fields should return err message", () => {
+      return request(app)
+        .post("/api/reviews/3/comments")
+        .send({})
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Invalid input");
+        });
+    });
+    it("400: failing schema validation", () => {
+      return request(app)
+        .post("/api/reviews/3/comments")
+        .send({ username: 123, body: 654 })
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Invalid input");
+        });
+    });
+  });
```

Comment:
It's still worth testing the usual errors here, 400 invalid review ID,  404 review ID not found etc. 

You should also include a test for 404 invalid username {username: "aUserThat IsntInTheDatabase", body : "validComment"}